shared/templates/create_services_disabled.py

```diff
@@ -4,19 +4,33 @@
 
 import sys
 
-from template_common import FilesGenerator, UnknownTargetError
+from template_common import FilesGenerator, UnknownTargetError, CSVLineError
 
 
 class ServiceDisabledGenerator(FilesGenerator):
     def generate(self, target, serviceinfo):
         try:
+            # mask service by default
+            mask_service = True
+
             # get the items out of the list
-            servicename, packagename, daemonname = serviceinfo
+            # items can be in format
+            # <service_name, package_name, daemon_name> or
+            # <service_name, package_name, daemon_name, dont_mask_service>
+            if len(serviceinfo) == 3:
+                servicename, packagename, daemonname = serviceinfo
+            elif len(serviceinfo) == 4:
+                servicename, packagename, daemonname, dont_mask_service = serviceinfo
+                # use boolean instead of any string that came from csv file
+                mask_service = False if dont_mask_service != "" else True
```

Comment:
I would advise strict interfaces, as they catch typos. So let's decide what values of the final argument are allowed, and let's enforce that.

Reply:
Restricted to `true` or `false`.